spec/models/organisation_spec.rb

```diff
@@ -84,4 +92,31 @@
       end
     end
   end
+
+  describe '#trim_name' do
+    it 'trims and converts name to upcase before save' do
+      organisation = build(:organisation, name_en: 'good city   ')
+      organisation.save
+      expect(organisation.reload.name_en).to eq('good city')
+    end
+  end
```

Comment:
I don't think we want to upcase before saving. Since organisations are coming from external/official sources we don't want to alter the data. We can just upcase during comparison cc @steveyken

Reply:
also the script that populates the data might bypass the the hooks which would end up with non-uppercase names anyways

Reply:
sorry about that 😅 
i need to change the test description. it doesn't convert it to upcase